### PR TITLE
tipAmout added to template

### LINKED ISSUE #2483: As a courier, I need to know if a tip has been given
Just to say thanks to the person who has given, avoiding to look ungrateful !

I need a tag of some sort on my app so I know.

### PR DESCRIPTION
**Pull request by AniaCuapio**
Fixes #2483

### CODE REVIEW

File: templates/order/summary.txt.twig [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{{ 'order.summary.heading'|trans({ '%number%': order.number, '%id%': order.id }) }}
+{{ 'order.summary.heading'|trans({ '%number%': order.number, '%id%': order.id , '%tipAmount%' order.tipAmount}) }}
```
Comment (alexsegura) [inline templates/order/summary.txt.twig:1]: Actually, it should be `order.getAdjustmentsTotal('tip')`, instead of `order.tipAmount`

Comment (alexsegura) [reply templates/order/summary.txt.twig:1]: You also need to modify the translation [order.summary.heading](https://github.com/coopcycle/coopcycle-web/blob/40d422d861c1d1c91a204003d1dec9652941c441/translations/messages.en.yml#L292) to show the tip amount

**Review by alexsegura — CHANGES_REQUESTED** [review]

**Comment by AniaCuapio** [comment]
Captura de pantalla de 2021-08-04 17-51-45

**Comment by AniaCuapio** [comment]
Maybe we can improve the way it is showed, but at least now it is showed.

**Review by alexsegura — APPROVED** [review]

**Pull request merged**